### PR TITLE
languagetool: disable specific rules

### PR DESCRIPTION
**Pull request by mateusz-bloch**
# Disabled rules: 
Examples why it's needed to disabled them:

ENGLISH_WORD_REPEAT_RULE :
image

RB_RB_COMMA[3]:
image

REP_PASSIVE_VOICE[6]:
image

AI[1]:
image

ID_CASING[2]:
image

ENGLISH_WORD_REPEAT_BEGINNING_RULE:
image

EN_UNPAIRED_BRACKETS:
image

### CODE REVIEW

**Review by damianloew — CHANGES_REQUESTED** [review]
As we discussed let's leave LC_AFTER_PERIOD and DATE_WEEKDAY_WITHOUT_YEAR

**Comment by nalajcie** [comment]
small note: please use imperative mood in commit messages (`disable` instead od `disabled`).

Worth considering: most of the issues stem from markdown code blocks being checked as native text - the ideal solution would be to make languagetool NOT check the code blocks at all (that way we could keep the rules enabled for native text). From quick research - I haven't found out-of-the box solution for it, but it probably could be done by LanguageTool API (so, writing our custom languagetool-based checker) and using it in CI.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -39,3 +39,4 @@ jobs:
         with:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review
+          disabled_rules: "ENGLISH_WORD_REPEAT_RULE,RB_RB_COMMA[3],REP_PASSIVE_VOICE[6],AI[1],ID_CASING[2],ENGLISH_WORD_REPEAT_BEGINNING_RULE,EN_UNPAIRED_BRACKETS"
```
Comment (damianloew) [inline .github/workflows/lint.yml:42]:
```suggestion
          disabled_rules: 
# TODO: make languagetool not check the code blocks and enable back most of rules mentioned below       "ENGLISH_WORD_REPEAT_RULE,RB_RB_COMMA[3],REP_PASSIVE_VOICE[6],AI[1],ID_CASING[2],ENGLISH_WORD_REPEAT_BEGINNING_RULE,EN_UNPAIRED_BRACKETS"
```

**Review by damianloew — COMMENTED** [review]
Please correct commit messages and leave the mentioned comment - IMHO we can merge it for now.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -39,3 +39,6 @@ jobs:
...
+          # TODO: make languagetool not check the code blocks and enable back most of rules mentioned below:
```
Comment (damianloew) [inline .github/workflows/lint.yml:43]:
```suggestion
          # TODO: make languagetool not check the code blocks and enable back rules mentioned below:
```

**Review by damianloew — APPROVED** [review]
LGTM

**Pull request merged**